architecture/core_concepts/routes.adoc

```diff
@@ -314,6 +320,47 @@ they are unique on the machine. These ports will not be exposed externally.
 
 The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
 
+[[load-balancing]]
+== Load-balancing Strategy
+
+When a route has multiple endpoints, HAProxy distributes requests to the route
+among the endpoints based on the selected load-balancing strategy. This only
+applies when no persistence information is available, or when a connection is
+redispatched to another server. See the discussion on
+xref:routes-sticky-sessions[Sticky Sessions]
+for more details. The strategy my one of the following:
+
+- `*roundrobin*`  Each server is used in turns, according to their weights.
+This is the smoothest and fairest algorithm when the server's
+processing time remains equally distributed. This algorithm
+is dynamic, which means that server weights may be adjusted
+on the fly for slow starts for instance. It is limited by
+design to 4 active servers per route.
```

Comment:
Servers or services?  I think services because the number of endpoints is not limited...